test/core/app.spec.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+
+console.log(testMode);
+
+describe("core.app", function () {
+
+  var app;
+
+  before(function() {
+    app = new Wu.App(systemapicConfigOptions);
+  });
+
+  it("Should check Wu version", function () {
```

Comment:
Very nice. but let's use the following format:

`it('should check Mapic.js version` ...)`

ie. "Should" with smallcaps ("should").

And instead of "Should test so and so", describe with the action of the test, for example `it('should have global app', ...)` and `it('should have Raven object)` etc...